TEZ-4300: Download protoc automatically compile/development time

abstractdog opened the pull request.

hadoop-yetus commented:
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |  13m 17s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +0 :ok: |  shelldocs  |   0m  1s |  Shelldocs was not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
| -1 :x: |  test4tests  |   0m  0s |  The patch doesn't appear to include any new or modified tests. Please justify why no new tests are needed for this patch. Also please list what manual steps were performed to verify this patch.  |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   6m  1s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   8m 21s |  master passed  |
| +1 :green_heart: |  compile  |   2m 46s |  master passed with JDK Ubuntu-11.0.10+9-Ubuntu-0ubuntu1.20.04  |
| +1 :green_heart: |  compile  |   2m 26s |  master passed with JDK Private Build-1.8.0_282-8u282-b08-0ubuntu1~20.04-b08  |
| +1 :green_heart: |  javadoc  |   2m 46s |  master passed with JDK Ubuntu-11.0.10+9-Ubuntu-0ubuntu1.20.04  |
| +1 :green_heart: |  javadoc  |   2m 15s |  master passed with JDK Private Build-1.8.0_282-8u282-b08-0ubuntu1~20.04-b08  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 18s |  Maven dependency ordering for patch  |
| -1 :x: |  mvninstall  |   0m  8s |  tez-api in the patch failed.  |
| -1 :x: |  mvninstall  |   1m 34s |  root in the patch failed.  |
| -1 :x: |  compile  |   0m 21s |  tez-api in the patch failed with JDK Ubuntu-11.0.10+9-Ubuntu-0ubuntu1.20.04.  |
| -1 :x: |  compile  |   0m 22s |  root in the patch failed with JDK Ubuntu-11.0.10+9-Ubuntu-0ubuntu1.20.04.  |
| -1 :x: |  javac  |   0m 21s |  tez-api in the patch failed with JDK Ubuntu-11.0.10+9-Ubuntu-0ubuntu1.20.04.  |
| -1 :x: |  javac  |   0m 22s |  root in the patch failed with JDK Ubuntu-11.0.10+9-Ubuntu-0ubuntu1.20.04.  |
| -1 :x: |  compile  |   0m 22s |  tez-api in the patch failed with JDK Private Build-1.8.0_282-8u282-b08-0ubuntu1~20.04-b08.  |
| -1 :x: |  compile  |   0m 20s |  root in the patch failed with JDK Private Build-1.8.0_282-8u282-b08-0ubuntu1~20.04-b08.  |
| -1 :x: |  javac  |   0m 22s |  tez-api in the patch failed with JDK Private Build-1.8.0_282-8u282-b08-0ubuntu1~20.04-b08.  |
| -1 :x: |  javac  |   0m 20s |  root in the patch failed with JDK Private Build-1.8.0_282-8u282-b08-0ubuntu1~20.04-b08.  |
| -1 :x: |  shellcheck  |   0m  1s |  The patch generated 22 new + 0 unchanged - 0 fixed = 22 total (was 0)  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  xml  |   0m  2s |  The patch has no ill-formed XML file.  |
| -1 :x: |  javadoc  |   0m 21s |  tez-api in the patch failed with JDK Ubuntu-11.0.10+9-Ubuntu-0ubuntu1.20.04.  |
| -1 :x: |  javadoc  |   0m 23s |  root in the patch failed with JDK Ubuntu-11.0.10+9-Ubuntu-0ubuntu1.20.04.  |
| -1 :x: |  javadoc  |   0m 20s |  tez-api in the patch failed with JDK Private Build-1.8.0_282-8u282-b08-0ubuntu1~20.04-b08.  |
| -1 :x: |  javadoc  |   0m 25s |  root in the patch failed with JDK Private Build-1.8.0_282-8u282-b08-0ubuntu1~20.04-b08.  |
||| _ Other Tests _ |
| -1 :x: |  unit  |   0m 18s |  tez-api in the patch failed.  |
| -1 :x: |  unit  |   2m  8s |  root in the patch failed.  |
| -1 :x: |  asflicense  |   0m 48s |  The patch generated 1 ASF License warnings.  |
|  |   |  47m 14s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/1/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/tez/pull/115 |
| JIRA Issue | TEZ-4300 |
| Optional Tests | dupname asflicense shellcheck shelldocs javac javadoc unit xml compile |
| uname | Linux 3ad936fc50d2 4.15.0-112-generic #113-Ubuntu SMP Thu Jul 9 23:41:39 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | personality/tez.sh |
| git revision | master / 2dcbe0bca |
| Default Java | Private Build-1.8.0_282-8u282-b08-0ubuntu1~20.04-b08 |
| Multi-JDK versions | /usr/lib/jvm/java-11-openjdk-amd64:Ubuntu-11.0.10+9-Ubuntu-0ubuntu1.20.04 /usr/lib/jvm/java-8-openjdk-amd64:Private Build-1.8.0_282-8u282-b08-0ubuntu1~20.04-b08 |
| mvninstall | https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/1/artifact/out/patch-mvninstall-tez-api.txt |
| mvninstall | https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/1/artifact/out/patch-mvninstall-root.txt |
| compile | https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/1/artifact/out/patch-compile-tez-api-jdkUbuntu-11.0.10+9-Ubuntu-0ubuntu1.20.04.txt |
| compile | https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/1/artifact/out/patch-compile-root-jdkUbuntu-11.0.10+9-Ubuntu-0ubuntu1.20.04.txt |
| javac | https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/1/artifact/out/patch-compile-tez-api-jdkUbuntu-11.0.10+9-Ubuntu-0ubuntu1.20.04.txt |
| javac | https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/1/artifact/out/patch-compile-root-jdkUbuntu-11.0.10+9-Ubuntu-0ubuntu1.20.04.txt |
| compile | https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/1/artifact/out/patch-compile-tez-api-jdkPrivateBuild-1.8.0_282-8u282-b08-0ubuntu1~20.04-b08.txt |
| compile | https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/1/artifact/out/patch-compile-root-jdkPrivateBuild-1.8.0_282-8u282-b08-0ubuntu1~20.04-b08.txt |
| javac | https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/1/artifact/out/patch-compile-tez-api-jdkPrivateBuild-1.8.0_282-8u282-b08-0ubuntu1~20.04-b08.txt |
| javac | https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/1/artifact/out/patch-compile-root-jdkPrivateBuild-1.8.0_282-8u282-b08-0ubuntu1~20.04-b08.txt |
| shellcheck | https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/1/artifact/out/diff-patch-shellcheck.txt |
| javadoc | https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/1/artifact/out/patch-javadoc-tez-api-jdkUbuntu-11.0.10+9-Ubuntu-0ubuntu1.20.04.txt |
| javadoc | https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/1/artifact/out/patch-javadoc-root-jdkUbuntu-11.0.10+9-Ubuntu-0ubuntu1.20.04.txt |
| javadoc | https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/1/artifact/out/patch-javadoc-tez-api-jdkPrivateBuild-1.8.0_282-8u282-b08-0ubuntu1~20.04-b08.txt |
| javadoc | https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/1/artifact/out/patch-javadoc-root-jdkPrivateBuild-1.8.0_282-8u282-b08-0ubuntu1~20.04-b08.txt |
| unit | https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/1/artifact/out/patch-unit-tez-api.txt |
| unit | https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/1/artifact/out/patch-unit-root.txt |
|  Test Results | https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/1/testReport/ |
| asflicense | https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/1/artifact/out/patch-asflicense-problems.txt |
| Max. process+thread count | 98 (vs. ulimit of 5500) |
| modules | C: tez-api . U: . |
| Console output | https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/1/console |
| versions | git=2.25.1 maven=3.6.3 shellcheck=0.7.1 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

hadoop-yetus commented:
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |  12m 50s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +0 :ok: |  shelldocs  |   0m  0s |  Shelldocs was not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
| -1 :x: |  test4tests  |   0m  0s |  The patch doesn't appear to include any new or modified tests. Please justify why no new tests are needed for this patch. Also please list what manual steps were performed to verify this patch.  |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   5m 43s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   8m 21s |  master passed  |
| +1 :green_heart: |  compile  |   2m 42s |  master passed with JDK Ubuntu-11.0.10+9-Ubuntu-0ubuntu1.20.04  |
| +1 :green_heart: |  compile  |   2m 30s |  master passed with JDK Private Build-1.8.0_282-8u282-b08-0ubuntu1~20.04-b08  |
| +1 :green_heart: |  javadoc  |   2m 44s |  master passed with JDK Ubuntu-11.0.10+9-Ubuntu-0ubuntu1.20.04  |
| +1 :green_heart: |  javadoc  |   2m 15s |  master passed with JDK Private Build-1.8.0_282-8u282-b08-0ubuntu1~20.04-b08  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 17s |  Maven dependency ordering for patch  |
| -1 :x: |  mvninstall  |   0m  8s |  tez-api in the patch failed.  |
| -1 :x: |  mvninstall  |   1m 35s |  root in the patch failed.  |
| -1 :x: |  compile  |   0m 23s |  tez-api in the patch failed with JDK Ubuntu-11.0.10+9-Ubuntu-0ubuntu1.20.04.  |
| -1 :x: |  compile  |   0m 24s |  root in the patch failed with JDK Ubuntu-11.0.10+9-Ubuntu-0ubuntu1.20.04.  |
| -1 :x: |  javac  |   0m 23s |  tez-api in the patch failed with JDK Ubuntu-11.0.10+9-Ubuntu-0ubuntu1.20.04.  |
| -1 :x: |  javac  |   0m 24s |  root in the patch failed with JDK Ubuntu-11.0.10+9-Ubuntu-0ubuntu1.20.04.  |
| -1 :x: |  compile  |   0m 22s |  tez-api in the patch failed with JDK Private Build-1.8.0_282-8u282-b08-0ubuntu1~20.04-b08.  |
| -1 :x: |  compile  |   0m 27s |  root in the patch failed with JDK Private Build-1.8.0_282-8u282-b08-0ubuntu1~20.04-b08.  |
| -1 :x: |  javac  |   0m 22s |  tez-api in the patch failed with JDK Private Build-1.8.0_282-8u282-b08-0ubuntu1~20.04-b08.  |
| -1 :x: |  javac  |   0m 27s |  root in the patch failed with JDK Private Build-1.8.0_282-8u282-b08-0ubuntu1~20.04-b08.  |
| -1 :x: |  shellcheck  |   0m  1s |  The patch generated 22 new + 0 unchanged - 0 fixed = 22 total (was 0)  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  xml  |   0m  3s |  The patch has no ill-formed XML file.  |
| -1 :x: |  javadoc  |   0m 21s |  tez-api in the patch failed with JDK Ubuntu-11.0.10+9-Ubuntu-0ubuntu1.20.04.  |
| -1 :x: |  javadoc  |   0m 25s |  root in the patch failed with JDK Ubuntu-11.0.10+9-Ubuntu-0ubuntu1.20.04.  |
| -1 :x: |  javadoc  |   0m 22s |  tez-api in the patch failed with JDK Private Build-1.8.0_282-8u282-b08-0ubuntu1~20.04-b08.  |
| -1 :x: |  javadoc  |   0m 25s |  root in the patch failed with JDK Private Build-1.8.0_282-8u282-b08-0ubuntu1~20.04-b08.  |
||| _ Other Tests _ |
| -1 :x: |  unit  |   0m 23s |  tez-api in the patch failed.  |
| -1 :x: |  unit  |   2m 13s |  root in the patch failed.  |
| -1 :x: |  asflicense  |   0m 54s |  The patch generated 1 ASF License warnings.  |
|  |   |  46m 55s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/2/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/tez/pull/115 |
| JIRA Issue | TEZ-4300 |
| Optional Tests | dupname asflicense shellcheck shelldocs javac javadoc unit xml compile |
| uname | Linux 0a3a8c1c2185 4.15.0-60-generic #67-Ubuntu SMP Thu Aug 22 16:55:30 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | personality/tez.sh |
| git revision | master / 2dcbe0bca |
| Default Java | Private Build-1.8.0_282-8u282-b08-0ubuntu1~20.04-b08 |
| Multi-JDK versions | /usr/lib/jvm/java-11-openjdk-amd64:Ubuntu-11.0.10+9-Ubuntu-0ubuntu1.20.04 /usr/lib/jvm/java-8-openjdk-amd64:Private Build-1.8.0_282-8u282-b08-0ubuntu1~20.04-b08 |
| mvninstall | https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/2/artifact/out/patch-mvninstall-tez-api.txt |
| mvninstall | https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/2/artifact/out/patch-mvninstall-root.txt |
| compile | https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/2/artifact/out/patch-compile-tez-api-jdkUbuntu-11.0.10+9-Ubuntu-0ubuntu1.20.04.txt |
| compile | https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/2/artifact/out/patch-compile-root-jdkUbuntu-11.0.10+9-Ubuntu-0ubuntu1.20.04.txt |
| javac | https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/2/artifact/out/patch-compile-tez-api-jdkUbuntu-11.0.10+9-Ubuntu-0ubuntu1.20.04.txt |
| javac | https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/2/artifact/out/patch-compile-root-jdkUbuntu-11.0.10+9-Ubuntu-0ubuntu1.20.04.txt |
| compile | https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/2/artifact/out/patch-compile-tez-api-jdkPrivateBuild-1.8.0_282-8u282-b08-0ubuntu1~20.04-b08.txt |
| compile | https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/2/artifact/out/patch-compile-root-jdkPrivateBuild-1.8.0_282-8u282-b08-0ubuntu1~20.04-b08.txt |
| javac | https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/2/artifact/out/patch-compile-tez-api-jdkPrivateBuild-1.8.0_282-8u282-b08-0ubuntu1~20.04-b08.txt |
| javac | https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/2/artifact/out/patch-compile-root-jdkPrivateBuild-1.8.0_282-8u282-b08-0ubuntu1~20.04-b08.txt |
| shellcheck | https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/2/artifact/out/diff-patch-shellcheck.txt |
| javadoc | https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/2/artifact/out/patch-javadoc-tez-api-jdkUbuntu-11.0.10+9-Ubuntu-0ubuntu1.20.04.txt |
| javadoc | https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/2/artifact/out/patch-javadoc-root-jdkUbuntu-11.0.10+9-Ubuntu-0ubuntu1.20.04.txt |
| javadoc | https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/2/artifact/out/patch-javadoc-tez-api-jdkPrivateBuild-1.8.0_282-8u282-b08-0ubuntu1~20.04-b08.txt |
| javadoc | https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/2/artifact/out/patch-javadoc-root-jdkPrivateBuild-1.8.0_282-8u282-b08-0ubuntu1~20.04-b08.txt |
| unit | https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/2/artifact/out/patch-unit-tez-api.txt |
| unit | https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/2/artifact/out/patch-unit-root.txt |
|  Test Results | https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/2/testReport/ |
| asflicense | https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/2/artifact/out/patch-asflicense-problems.txt |
| Max. process+thread count | 99 (vs. ulimit of 5500) |
| modules | C: tez-api . U: . |
| Console output | https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/2/console |
| versions | git=2.25.1 maven=3.6.3 shellcheck=0.7.1 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

hadoop-yetus commented:
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |  17m 23s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +0 :ok: |  shelldocs  |   0m  0s |  Shelldocs was not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
| -1 :x: |  test4tests  |   0m  0s |  The patch doesn't appear to include any new or modified tests. Please justify why no new tests are needed for this patch. Also please list what manual steps were performed to verify this patch.  |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   5m 57s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   8m 38s |  master passed  |
| +1 :green_heart: |  compile  |   3m  1s |  master passed with JDK Ubuntu-11.0.10+9-Ubuntu-0ubuntu1.20.04  |
| +1 :green_heart: |  compile  |   3m 25s |  master passed with JDK Private Build-1.8.0_282-8u282-b08-0ubuntu1~20.04-b08  |
| +1 :green_heart: |  javadoc  |   3m 40s |  master passed with JDK Ubuntu-11.0.10+9-Ubuntu-0ubuntu1.20.04  |
| +1 :green_heart: |  javadoc  |   3m  3s |  master passed with JDK Private Build-1.8.0_282-8u282-b08-0ubuntu1~20.04-b08  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 21s |  Maven dependency ordering for patch  |
| -1 :x: |  mvninstall  |   0m 10s |  tez-api in the patch failed.  |
| -1 :x: |  compile  |   0m 20s |  tez-api in the patch failed with JDK Ubuntu-11.0.10+9-Ubuntu-0ubuntu1.20.04.  |
| -1 :x: |  javac  |   0m 20s |  tez-api in the patch failed with JDK Ubuntu-11.0.10+9-Ubuntu-0ubuntu1.20.04.  |
| -1 :x: |  compile  |   0m 20s |  tez-api in the patch failed with JDK Private Build-1.8.0_282-8u282-b08-0ubuntu1~20.04-b08.  |
| -1 :x: |  javac  |   0m 20s |  tez-api in the patch failed with JDK Private Build-1.8.0_282-8u282-b08-0ubuntu1~20.04-b08.  |
| -1 :x: |  shellcheck  |   0m  0s |  The patch generated 17 new + 0 unchanged - 0 fixed = 17 total (was 0)  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  xml  |   0m  3s |  The patch has no ill-formed XML file.  |
| -1 :x: |  javadoc  |   0m 19s |  tez-api in the patch failed with JDK Ubuntu-11.0.10+9-Ubuntu-0ubuntu1.20.04.  |
| -1 :x: |  javadoc  |   0m 19s |  tez-api in the patch failed with JDK Private Build-1.8.0_282-8u282-b08-0ubuntu1~20.04-b08.  |
||| _ Other Tests _ |
| -1 :x: |  unit  |   0m 18s |  tez-api in the patch failed.  |
| +1 :green_heart: |  unit  |  76m 55s |  root in the patch passed.  |
| -1 :x: |  asflicense  |   0m 55s |  The patch generated 1 ASF License warnings.  |
|  |   | 141m  4s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/3/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/tez/pull/115 |
| JIRA Issue | TEZ-4300 |
| Optional Tests | dupname asflicense shellcheck shelldocs javac javadoc unit xml compile |
| uname | Linux ccfdaf8664d4 4.15.0-136-generic #140-Ubuntu SMP Thu Jan 28 05:20:47 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | personality/tez.sh |
| git revision | master / 2dcbe0bca |
| Default Java | Private Build-1.8.0_282-8u282-b08-0ubuntu1~20.04-b08 |
| Multi-JDK versions | /usr/lib/jvm/java-11-openjdk-amd64:Ubuntu-11.0.10+9-Ubuntu-0ubuntu1.20.04 /usr/lib/jvm/java-8-openjdk-amd64:Private Build-1.8.0_282-8u282-b08-0ubuntu1~20.04-b08 |
| mvninstall | https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/3/artifact/out/patch-mvninstall-tez-api.txt |
| compile | https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/3/artifact/out/patch-compile-tez-api-jdkUbuntu-11.0.10+9-Ubuntu-0ubuntu1.20.04.txt |
| javac | https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/3/artifact/out/patch-compile-tez-api-jdkUbuntu-11.0.10+9-Ubuntu-0ubuntu1.20.04.txt |
| compile | https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/3/artifact/out/patch-compile-tez-api-jdkPrivateBuild-1.8.0_282-8u282-b08-0ubuntu1~20.04-b08.txt |
| javac | https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/3/artifact/out/patch-compile-tez-api-jdkPrivateBuild-1.8.0_282-8u282-b08-0ubuntu1~20.04-b08.txt |
| shellcheck | https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/3/artifact/out/diff-patch-shellcheck.txt |
| javadoc | https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/3/artifact/out/patch-javadoc-tez-api-jdkUbuntu-11.0.10+9-Ubuntu-0ubuntu1.20.04.txt |
| javadoc | https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/3/artifact/out/patch-javadoc-tez-api-jdkPrivateBuild-1.8.0_282-8u282-b08-0ubuntu1~20.04-b08.txt |
| unit | https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/3/artifact/out/patch-unit-tez-api.txt |
|  Test Results | https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/3/testReport/ |
| asflicense | https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/3/artifact/out/patch-asflicense-problems.txt |
| Max. process+thread count | 1520 (vs. ulimit of 5500) |
| modules | C: tez-api . U: . |
| Console output | https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/3/console |
| versions | git=2.25.1 maven=3.6.3 shellcheck=0.7.1 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

hadoop-yetus commented:
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   0m 51s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +0 :ok: |  shelldocs  |   0m  0s |  Shelldocs was not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
| -1 :x: |  test4tests  |   0m  0s |  The patch doesn't appear to include any new or modified tests. Please justify why no new tests are needed for this patch. Also please list what manual steps were performed to verify this patch.  |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   6m  8s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   9m 32s |  master passed  |
| +1 :green_heart: |  compile  |   3m 27s |  master passed with JDK Ubuntu-11.0.10+9-Ubuntu-0ubuntu1.20.04  |
| +1 :green_heart: |  compile  |   2m 56s |  master passed with JDK Private Build-1.8.0_282-8u282-b08-0ubuntu1~20.04-b08  |
| +1 :green_heart: |  javadoc  |   3m 11s |  master passed with JDK Ubuntu-11.0.10+9-Ubuntu-0ubuntu1.20.04  |
| +1 :green_heart: |  javadoc  |   2m 27s |  master passed with JDK Private Build-1.8.0_282-8u282-b08-0ubuntu1~20.04-b08  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 19s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   5m  7s |  the patch passed  |
| +1 :green_heart: |  compile  |   3m 34s |  the patch passed with JDK Ubuntu-11.0.10+9-Ubuntu-0ubuntu1.20.04  |
| +1 :green_heart: |  javac  |   3m 34s |  the patch passed  |
| +1 :green_heart: |  compile  |   3m 10s |  the patch passed with JDK Private Build-1.8.0_282-8u282-b08-0ubuntu1~20.04-b08  |
| +1 :green_heart: |  javac  |   3m 10s |  the patch passed  |
| -1 :x: |  shellcheck  |   0m  0s |  The patch generated 3 new + 0 unchanged - 0 fixed = 3 total (was 0)  |
| -1 :x: |  whitespace  |   0m  0s |  The patch has 1 line(s) that end in whitespace. Use git apply --whitespace=fix <<patch_file>>. Refer https://git-scm.com/docs/git-apply  |
| +1 :green_heart: |  xml  |   0m  4s |  The patch has no ill-formed XML file.  |
| +1 :green_heart: |  javadoc  |   3m 10s |  the patch passed with JDK Ubuntu-11.0.10+9-Ubuntu-0ubuntu1.20.04  |
| +1 :green_heart: |  javadoc  |   2m 29s |  the patch passed with JDK Private Build-1.8.0_282-8u282-b08-0ubuntu1~20.04-b08  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  |   2m 14s |  tez-api in the patch passed.  |
| +1 :green_heart: |  unit  |  73m 41s |  root in the patch passed.  |
| -1 :x: |  asflicense  |   1m  5s |  The patch generated 1 ASF License warnings.  |
|  |   | 124m 39s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/4/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/tez/pull/115 |
| JIRA Issue | TEZ-4300 |
| Optional Tests | dupname asflicense shellcheck shelldocs javac javadoc unit xml compile |
| uname | Linux 1f8ea6dcf0df 4.15.0-112-generic #113-Ubuntu SMP Thu Jul 9 23:41:39 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | personality/tez.sh |
| git revision | master / 2dcbe0bca |
| Default Java | Private Build-1.8.0_282-8u282-b08-0ubuntu1~20.04-b08 |
| Multi-JDK versions | /usr/lib/jvm/java-11-openjdk-amd64:Ubuntu-11.0.10+9-Ubuntu-0ubuntu1.20.04 /usr/lib/jvm/java-8-openjdk-amd64:Private Build-1.8.0_282-8u282-b08-0ubuntu1~20.04-b08 |
| shellcheck | https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/4/artifact/out/diff-patch-shellcheck.txt |
| whitespace | https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/4/artifact/out/whitespace-eol.txt |
|  Test Results | https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/4/testReport/ |
| asflicense | https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/4/artifact/out/patch-asflicense-problems.txt |
| Max. process+thread count | 1664 (vs. ulimit of 5500) |
| modules | C: tez-api . U: . |
| Console output | https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/4/console |
| versions | git=2.25.1 maven=3.6.3 shellcheck=0.7.1 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

hadoop-yetus commented:
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   1m 40s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +0 :ok: |  shelldocs  |   0m  0s |  Shelldocs was not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
| -1 :x: |  test4tests  |   0m  0s |  The patch doesn't appear to include any new or modified tests. Please justify why no new tests are needed for this patch. Also please list what manual steps were performed to verify this patch.  |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   5m 20s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   9m 52s |  master passed  |
| +1 :green_heart: |  compile  |   3m  9s |  master passed with JDK Ubuntu-11.0.10+9-Ubuntu-0ubuntu1.20.04  |
| +1 :green_heart: |  compile  |   2m 57s |  master passed with JDK Private Build-1.8.0_282-8u282-b08-0ubuntu1~20.04-b08  |
| +1 :green_heart: |  javadoc  |   3m 26s |  master passed with JDK Ubuntu-11.0.10+9-Ubuntu-0ubuntu1.20.04  |
| +1 :green_heart: |  javadoc  |   2m 45s |  master passed with JDK Private Build-1.8.0_282-8u282-b08-0ubuntu1~20.04-b08  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 13s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   5m  6s |  the patch passed  |
| +1 :green_heart: |  compile  |   3m 35s |  the patch passed with JDK Ubuntu-11.0.10+9-Ubuntu-0ubuntu1.20.04  |
| +1 :green_heart: |  javac  |   3m 35s |  the patch passed  |
| +1 :green_heart: |  compile  |   2m 55s |  the patch passed with JDK Private Build-1.8.0_282-8u282-b08-0ubuntu1~20.04-b08  |
| +1 :green_heart: |  javac  |   2m 55s |  the patch passed  |
| -1 :x: |  shellcheck  |   0m  1s |  The patch generated 3 new + 0 unchanged - 0 fixed = 3 total (was 0)  |
| -1 :x: |  whitespace  |   0m  0s |  The patch has 1 line(s) that end in whitespace. Use git apply --whitespace=fix <<patch_file>>. Refer https://git-scm.com/docs/git-apply  |
| +1 :green_heart: |  xml  |   0m  1s |  The patch has no ill-formed XML file.  |
| +1 :green_heart: |  javadoc  |   3m 11s |  the patch passed with JDK Ubuntu-11.0.10+9-Ubuntu-0ubuntu1.20.04  |
| +1 :green_heart: |  javadoc  |   2m 37s |  the patch passed with JDK Private Build-1.8.0_282-8u282-b08-0ubuntu1~20.04-b08  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  |   2m 17s |  tez-api in the patch passed.  |
| +1 :green_heart: |  unit  |  74m 22s |  root in the patch passed.  |
| -1 :x: |  asflicense  |   0m 51s |  The patch generated 1 ASF License warnings.  |
|  |   | 125m 30s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/5/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/tez/pull/115 |
| JIRA Issue | TEZ-4300 |
| Optional Tests | dupname asflicense shellcheck shelldocs javac javadoc unit xml compile |
| uname | Linux dc358cd71718 4.15.0-136-generic #140-Ubuntu SMP Thu Jan 28 05:20:47 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | personality/tez.sh |
| git revision | master / 2dcbe0bca |
| Default Java | Private Build-1.8.0_282-8u282-b08-0ubuntu1~20.04-b08 |
| Multi-JDK versions | /usr/lib/jvm/java-11-openjdk-amd64:Ubuntu-11.0.10+9-Ubuntu-0ubuntu1.20.04 /usr/lib/jvm/java-8-openjdk-amd64:Private Build-1.8.0_282-8u282-b08-0ubuntu1~20.04-b08 |
| shellcheck | https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/5/artifact/out/diff-patch-shellcheck.txt |
| whitespace | https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/5/artifact/out/whitespace-eol.txt |
|  Test Results | https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/5/testReport/ |
| asflicense | https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/5/artifact/out/patch-asflicense-problems.txt |
| Max. process+thread count | 1546 (vs. ulimit of 5500) |
| modules | C: tez-api . U: . |
| Console output | https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/5/console |
| versions | git=2.25.1 maven=3.6.3 shellcheck=0.7.1 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

hadoop-yetus commented:
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   0m 34s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +0 :ok: |  shelldocs  |   0m  0s |  Shelldocs was not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
| -1 :x: |  test4tests  |   0m  0s |  The patch doesn't appear to include any new or modified tests. Please justify why no new tests are needed for this patch. Also please list what manual steps were performed to verify this patch.  |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   5m 10s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   8m 49s |  master passed  |
| +1 :green_heart: |  compile  |   2m 39s |  master passed with JDK Ubuntu-11.0.10+9-Ubuntu-0ubuntu1.20.04  |
| +1 :green_heart: |  compile  |   2m 28s |  master passed with JDK Private Build-1.8.0_282-8u282-b08-0ubuntu1~20.04-b08  |
| +1 :green_heart: |  javadoc  |   2m 46s |  master passed with JDK Ubuntu-11.0.10+9-Ubuntu-0ubuntu1.20.04  |
| +1 :green_heart: |  javadoc  |   2m 13s |  master passed with JDK Private Build-1.8.0_282-8u282-b08-0ubuntu1~20.04-b08  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 12s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   3m 56s |  the patch passed  |
| +1 :green_heart: |  compile  |   2m 41s |  the patch passed with JDK Ubuntu-11.0.10+9-Ubuntu-0ubuntu1.20.04  |
| +1 :green_heart: |  javac  |   2m 41s |  the patch passed  |
| +1 :green_heart: |  compile  |   2m 26s |  the patch passed with JDK Private Build-1.8.0_282-8u282-b08-0ubuntu1~20.04-b08  |
| +1 :green_heart: |  javac  |   2m 26s |  the patch passed  |
| -1 :x: |  shellcheck  |   0m  0s |  The patch generated 1 new + 0 unchanged - 0 fixed = 1 total (was 0)  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  xml  |   0m  1s |  The patch has no ill-formed XML file.  |
| +1 :green_heart: |  javadoc  |   2m 34s |  the patch passed with JDK Ubuntu-11.0.10+9-Ubuntu-0ubuntu1.20.04  |
| +1 :green_heart: |  javadoc  |   2m 14s |  the patch passed with JDK Private Build-1.8.0_282-8u282-b08-0ubuntu1~20.04-b08  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  |   2m 10s |  tez-api in the patch passed.  |
| -1 :x: |  unit  |  60m 29s |  root in the patch failed.  |
| +1 :green_heart: |  asflicense  |   1m  1s |  The patch does not generate ASF License warnings.  |
|  |   | 103m 33s |   |


| Reason | Tests |
|-------:|:------|
| Failed junit tests | tez.test.TestRecovery |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/6/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/tez/pull/115 |
| JIRA Issue | TEZ-4300 |
| Optional Tests | dupname asflicense shellcheck shelldocs javac javadoc unit xml compile |
| uname | Linux 17e01f0a5a13 4.15.0-112-generic #113-Ubuntu SMP Thu Jul 9 23:41:39 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | personality/tez.sh |
| git revision | master / 2dcbe0bca |
| Default Java | Private Build-1.8.0_282-8u282-b08-0ubuntu1~20.04-b08 |
| Multi-JDK versions | /usr/lib/jvm/java-11-openjdk-amd64:Ubuntu-11.0.10+9-Ubuntu-0ubuntu1.20.04 /usr/lib/jvm/java-8-openjdk-amd64:Private Build-1.8.0_282-8u282-b08-0ubuntu1~20.04-b08 |
| shellcheck | https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/6/artifact/out/diff-patch-shellcheck.txt |
| unit | https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/6/artifact/out/patch-unit-root.txt |
|  Test Results | https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/6/testReport/ |
| Max. process+thread count | 1242 (vs. ulimit of 5500) |
| modules | C: tez-api . U: . |
| Console output | https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/6/console |
| versions | git=2.25.1 maven=3.6.3 shellcheck=0.7.1 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

tez-yetus commented:
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |  16m  4s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +0 :ok: |  shelldocs  |   0m  0s |  Shelldocs was not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
| -1 :x: |  test4tests  |   0m  0s |  The patch doesn't appear to include any new or modified tests. Please justify why no new tests are needed for this patch. Also please list what manual steps were performed to verify this patch.  |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   4m 26s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   9m 24s |  master passed  |
| +1 :green_heart: |  compile  |   2m 45s |  master passed with JDK Ubuntu-11.0.13+8-Ubuntu-0ubuntu1.20.04  |
| +1 :green_heart: |  compile  |   2m 33s |  master passed with JDK Private Build-1.8.0_312-8u312-b07-0ubuntu1~20.04-b07  |
| +1 :green_heart: |  javadoc  |   2m 41s |  master passed with JDK Ubuntu-11.0.13+8-Ubuntu-0ubuntu1.20.04  |
| +1 :green_heart: |  javadoc  |   2m  8s |  master passed with JDK Private Build-1.8.0_312-8u312-b07-0ubuntu1~20.04-b07  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 12s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   4m 25s |  the patch passed  |
| +1 :green_heart: |  compile  |   2m 48s |  the patch passed with JDK Ubuntu-11.0.13+8-Ubuntu-0ubuntu1.20.04  |
| +1 :green_heart: |  javac  |   2m 48s |  the patch passed  |
| +1 :green_heart: |  compile  |   2m 31s |  the patch passed with JDK Private Build-1.8.0_312-8u312-b07-0ubuntu1~20.04-b07  |
| +1 :green_heart: |  javac  |   2m 31s |  the patch passed  |
| +1 :green_heart: |  shellcheck  |   0m  1s |  There were no new shellcheck issues.  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  xml  |   0m  1s |  The patch has no ill-formed XML file.  |
| +1 :green_heart: |  javadoc  |   2m 33s |  the patch passed with JDK Ubuntu-11.0.13+8-Ubuntu-0ubuntu1.20.04  |
| +1 :green_heart: |  javadoc  |   2m  8s |  the patch passed with JDK Private Build-1.8.0_312-8u312-b07-0ubuntu1~20.04-b07  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  |   2m  4s |  tez-api in the patch passed.  |
| -1 :x: |  unit  |  70m 17s |  root in the patch failed.  |
| +1 :green_heart: |  asflicense  |   0m 48s |  The patch does not generate ASF License warnings.  |
|  |   | 128m 55s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/7/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/tez/pull/115 |
| JIRA Issue | TEZ-4300 |
| Optional Tests | dupname asflicense shellcheck shelldocs javac javadoc unit xml compile |
| uname | Linux b4a5fb5b7444 4.15.0-163-generic #171-Ubuntu SMP Fri Nov 5 11:55:11 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | personality/tez.sh |
| git revision | master / bf616b220 |
| Default Java | Private Build-1.8.0_312-8u312-b07-0ubuntu1~20.04-b07 |
| Multi-JDK versions | /usr/lib/jvm/java-11-openjdk-amd64:Ubuntu-11.0.13+8-Ubuntu-0ubuntu1.20.04 /usr/lib/jvm/java-8-openjdk-amd64:Private Build-1.8.0_312-8u312-b07-0ubuntu1~20.04-b07 |
| unit | https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/7/artifact/out/patch-unit-root.txt |
|  Test Results | https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/7/testReport/ |
| Max. process+thread count | 2089 (vs. ulimit of 5500) |
| modules | C: tez-api . U: . |
| Console output | https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/7/console |
| versions | git=2.25.1 maven=3.6.3 shellcheck=0.7.1 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

tez-yetus commented:
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   1m  6s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  1s |  No case conflicting files found.  |
| +0 :ok: |  shelldocs  |   0m  1s |  Shelldocs was not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
| -1 :x: |  test4tests  |   0m  0s |  The patch doesn't appear to include any new or modified tests. Please justify why no new tests are needed for this patch. Also please list what manual steps were performed to verify this patch.  |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   4m 24s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   9m 22s |  master passed  |
| +1 :green_heart: |  compile  |   2m 48s |  master passed with JDK Ubuntu-11.0.13+8-Ubuntu-0ubuntu1.20.04  |
| +1 :green_heart: |  compile  |   2m 30s |  master passed with JDK Private Build-1.8.0_312-8u312-b07-0ubuntu1~20.04-b07  |
| +1 :green_heart: |  javadoc  |   2m 41s |  master passed with JDK Ubuntu-11.0.13+8-Ubuntu-0ubuntu1.20.04  |
| +1 :green_heart: |  javadoc  |   2m  7s |  master passed with JDK Private Build-1.8.0_312-8u312-b07-0ubuntu1~20.04-b07  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 11s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   4m 25s |  the patch passed  |
| +1 :green_heart: |  compile  |   2m 45s |  the patch passed with JDK Ubuntu-11.0.13+8-Ubuntu-0ubuntu1.20.04  |
| +1 :green_heart: |  javac  |   2m 45s |  the patch passed  |
| +1 :green_heart: |  compile  |   2m 31s |  the patch passed with JDK Private Build-1.8.0_312-8u312-b07-0ubuntu1~20.04-b07  |
| +1 :green_heart: |  javac  |   2m 30s |  the patch passed  |
| +1 :green_heart: |  shellcheck  |   0m  0s |  There were no new shellcheck issues.  |
| +1 :green_heart: |  whitespace  |   0m  1s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  xml  |   0m  1s |  The patch has no ill-formed XML file.  |
| +1 :green_heart: |  javadoc  |   2m 31s |  the patch passed with JDK Ubuntu-11.0.13+8-Ubuntu-0ubuntu1.20.04  |
| +1 :green_heart: |  javadoc  |   2m 11s |  the patch passed with JDK Private Build-1.8.0_312-8u312-b07-0ubuntu1~20.04-b07  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  |   2m  4s |  tez-api in the patch passed.  |
| -1 :x: |  unit  |  69m 11s |  root in the patch failed.  |
| +1 :green_heart: |  asflicense  |   0m 49s |  The patch does not generate ASF License warnings.  |
|  |   | 112m 39s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/8/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/tez/pull/115 |
| JIRA Issue | TEZ-4300 |
| Optional Tests | dupname asflicense shellcheck shelldocs javac javadoc unit xml compile |
| uname | Linux 53560040ebec 4.15.0-163-generic #171-Ubuntu SMP Fri Nov 5 11:55:11 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | personality/tez.sh |
| git revision | master / c9b8e90db |
| Default Java | Private Build-1.8.0_312-8u312-b07-0ubuntu1~20.04-b07 |
| Multi-JDK versions | /usr/lib/jvm/java-11-openjdk-amd64:Ubuntu-11.0.13+8-Ubuntu-0ubuntu1.20.04 /usr/lib/jvm/java-8-openjdk-amd64:Private Build-1.8.0_312-8u312-b07-0ubuntu1~20.04-b07 |
| unit | https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/8/artifact/out/patch-unit-root.txt |
|  Test Results | https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/8/testReport/ |
| Max. process+thread count | 2089 (vs. ulimit of 5500) |
| modules | C: tez-api . U: . |
| Console output | https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/8/console |
| versions | git=2.25.1 maven=3.6.3 shellcheck=0.7.1 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

abstractdog commented:
what do you think about this @jteagles? this would help with automatically handling different protoc versions, driven only by protobuf.version property

jteagles commented:
@abstractdog, As a project, is this needed only for Hadoop 3.x projects or is there value in branch-0.9 hadoop 2.x based projects?

abstractdog commented:
it makes sense to try the patch out on branch-0.9 as it's only about using protobuf inside tez, let me try

abstractdog commented:
checked, the patch works on branch-0.9 too, [this commit](https://github.com/abstractdog/tez/commit/f0f9923d782be9e4f6a0fedaae9f6f5103f89f06)

contains a fix that I added on master too, please refer to the second commit: moved the profile to root pom.xml + redefined where it was needed

is this okay to go in?

tez-yetus commented:
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |  16m 17s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +0 :ok: |  shelldocs  |   0m  0s |  Shelldocs was not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
| -1 :x: |  test4tests  |   0m  0s |  The patch doesn't appear to include any new or modified tests. Please justify why no new tests are needed for this patch. Also please list what manual steps were performed to verify this patch.  |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   5m 36s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   8m 56s |  master passed  |
| +1 :green_heart: |  compile  |   3m 12s |  master passed with JDK Ubuntu-11.0.13+8-Ubuntu-0ubuntu1.20.04  |
| +1 :green_heart: |  compile  |   2m 54s |  master passed with JDK Private Build-1.8.0_312-8u312-b07-0ubuntu1~20.04-b07  |
| +1 :green_heart: |  javadoc  |   3m  8s |  master passed with JDK Ubuntu-11.0.13+8-Ubuntu-0ubuntu1.20.04  |
| +1 :green_heart: |  javadoc  |   2m 29s |  master passed with JDK Private Build-1.8.0_312-8u312-b07-0ubuntu1~20.04-b07  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 15s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   4m 38s |  the patch passed  |
| +1 :green_heart: |  compile  |   3m 12s |  the patch passed with JDK Ubuntu-11.0.13+8-Ubuntu-0ubuntu1.20.04  |
| +1 :green_heart: |  javac  |   3m 12s |  the patch passed  |
| +1 :green_heart: |  compile  |   2m 54s |  the patch passed with JDK Private Build-1.8.0_312-8u312-b07-0ubuntu1~20.04-b07  |
| +1 :green_heart: |  javac  |   2m 54s |  the patch passed  |
| +1 :green_heart: |  shellcheck  |   0m  0s |  There were no new shellcheck issues.  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  xml  |   0m  3s |  The patch has no ill-formed XML file.  |
| +1 :green_heart: |  javadoc  |   2m 53s |  the patch passed with JDK Ubuntu-11.0.13+8-Ubuntu-0ubuntu1.20.04  |
| +1 :green_heart: |  javadoc  |   2m 28s |  the patch passed with JDK Private Build-1.8.0_312-8u312-b07-0ubuntu1~20.04-b07  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  |   2m  5s |  tez-api in the patch passed.  |
| +1 :green_heart: |  unit  |   0m 31s |  tez-protobuf-history-plugin in the patch passed.  |
| +1 :green_heart: |  unit  |  72m 36s |  root in the patch passed.  |
| +1 :green_heart: |  asflicense  |   1m  8s |  The patch does not generate ASF License warnings.  |
|  |   | 136m 24s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/9/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/tez/pull/115 |
| JIRA Issue | TEZ-4300 |
| Optional Tests | dupname asflicense shellcheck shelldocs javac javadoc unit xml compile |
| uname | Linux 75aebc9c6c62 4.15.0-163-generic #171-Ubuntu SMP Fri Nov 5 11:55:11 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | personality/tez.sh |
| git revision | master / 227c75014 |
| Default Java | Private Build-1.8.0_312-8u312-b07-0ubuntu1~20.04-b07 |
| Multi-JDK versions | /usr/lib/jvm/java-11-openjdk-amd64:Ubuntu-11.0.13+8-Ubuntu-0ubuntu1.20.04 /usr/lib/jvm/java-8-openjdk-amd64:Private Build-1.8.0_312-8u312-b07-0ubuntu1~20.04-b07 |
|  Test Results | https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/9/testReport/ |
| Max. process+thread count | 1387 (vs. ulimit of 5500) |
| modules | C: tez-api tez-plugins/tez-protobuf-history-plugin . U: . |
| Console output | https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/9/console |
| versions | git=2.25.1 maven=3.6.3 shellcheck=0.7.1 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

abstractdog commented:
@jteagles : can I merge this to master? and push it to branch-0.9

tez-yetus commented:
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   1m 24s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +0 :ok: |  shelldocs  |   0m  1s |  Shelldocs was not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
| -1 :x: |  test4tests  |   0m  0s |  The patch doesn't appear to include any new or modified tests. Please justify why no new tests are needed for this patch. Also please list what manual steps were performed to verify this patch.  |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   5m 40s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |  10m  5s |  master passed  |
| +1 :green_heart: |  compile  |   3m 33s |  master passed with JDK Ubuntu-11.0.13+8-Ubuntu-0ubuntu1.20.04  |
| +1 :green_heart: |  compile  |   3m 14s |  master passed with JDK Private Build-1.8.0_312-8u312-b07-0ubuntu1~20.04-b07  |
| +1 :green_heart: |  javadoc  |   3m 20s |  master passed with JDK Ubuntu-11.0.13+8-Ubuntu-0ubuntu1.20.04  |
| +1 :green_heart: |  javadoc  |   2m 41s |  master passed with JDK Private Build-1.8.0_312-8u312-b07-0ubuntu1~20.04-b07  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 18s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   5m 21s |  the patch passed  |
| +1 :green_heart: |  compile  |   3m 24s |  the patch passed with JDK Ubuntu-11.0.13+8-Ubuntu-0ubuntu1.20.04  |
| +1 :green_heart: |  javac  |   3m 24s |  the patch passed  |
| +1 :green_heart: |  compile  |   3m  5s |  the patch passed with JDK Private Build-1.8.0_312-8u312-b07-0ubuntu1~20.04-b07  |
| +1 :green_heart: |  javac  |   3m  5s |  the patch passed  |
| +1 :green_heart: |  shellcheck  |   0m  0s |  There were no new shellcheck issues.  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  xml  |   0m  4s |  The patch has no ill-formed XML file.  |
| +1 :green_heart: |  javadoc  |   3m  1s |  the patch passed with JDK Ubuntu-11.0.13+8-Ubuntu-0ubuntu1.20.04  |
| +1 :green_heart: |  javadoc  |   2m 32s |  the patch passed with JDK Private Build-1.8.0_312-8u312-b07-0ubuntu1~20.04-b07  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  |   2m 11s |  tez-api in the patch passed.  |
| +1 :green_heart: |  unit  |   0m 34s |  tez-protobuf-history-plugin in the patch passed.  |
| +1 :green_heart: |  unit  |  69m 25s |  root in the patch passed.  |
| +1 :green_heart: |  asflicense  |   1m 19s |  The patch does not generate ASF License warnings.  |
|  |   | 122m 28s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/10/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/tez/pull/115 |
| JIRA Issue | TEZ-4300 |
| Optional Tests | dupname asflicense shellcheck shelldocs javac javadoc unit xml compile |
| uname | Linux f8ee9d6a20f2 4.15.0-163-generic #171-Ubuntu SMP Fri Nov 5 11:55:11 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | personality/tez.sh |
| git revision | master / 6d7ef203c |
| Default Java | Private Build-1.8.0_312-8u312-b07-0ubuntu1~20.04-b07 |
| Multi-JDK versions | /usr/lib/jvm/java-11-openjdk-amd64:Ubuntu-11.0.13+8-Ubuntu-0ubuntu1.20.04 /usr/lib/jvm/java-8-openjdk-amd64:Private Build-1.8.0_312-8u312-b07-0ubuntu1~20.04-b07 |
|  Test Results | https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/10/testReport/ |
| Max. process+thread count | 1399 (vs. ulimit of 5500) |
| modules | C: tez-api tez-plugins/tez-protobuf-history-plugin . U: . |
| Console output | https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/10/console |
| versions | git=2.25.1 maven=3.6.3 shellcheck=0.7.1 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

tez-yetus commented:
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |  16m 53s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +0 :ok: |  shelldocs  |   0m  0s |  Shelldocs was not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
| -1 :x: |  test4tests  |   0m  0s |  The patch doesn't appear to include any new or modified tests. Please justify why no new tests are needed for this patch. Also please list what manual steps were performed to verify this patch.  |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   5m 21s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   9m 16s |  master passed  |
| +1 :green_heart: |  compile  |   3m 19s |  master passed with JDK Ubuntu-11.0.13+8-Ubuntu-0ubuntu1.20.04  |
| +1 :green_heart: |  compile  |   3m  2s |  master passed with JDK Private Build-1.8.0_312-8u312-b07-0ubuntu1~20.04-b07  |
| +1 :green_heart: |  javadoc  |   3m  4s |  master passed with JDK Ubuntu-11.0.13+8-Ubuntu-0ubuntu1.20.04  |
| +1 :green_heart: |  javadoc  |   2m 29s |  master passed with JDK Private Build-1.8.0_312-8u312-b07-0ubuntu1~20.04-b07  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 15s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   4m 59s |  the patch passed  |
| +1 :green_heart: |  compile  |   3m 18s |  the patch passed with JDK Ubuntu-11.0.13+8-Ubuntu-0ubuntu1.20.04  |
| +1 :green_heart: |  javac  |   3m 18s |  the patch passed  |
| +1 :green_heart: |  compile  |   3m  2s |  the patch passed with JDK Private Build-1.8.0_312-8u312-b07-0ubuntu1~20.04-b07  |
| +1 :green_heart: |  javac  |   3m  2s |  the patch passed  |
| +1 :green_heart: |  shellcheck  |   0m  0s |  There were no new shellcheck issues.  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  xml  |   0m  3s |  The patch has no ill-formed XML file.  |
| +1 :green_heart: |  javadoc  |   2m 53s |  the patch passed with JDK Ubuntu-11.0.13+8-Ubuntu-0ubuntu1.20.04  |
| +1 :green_heart: |  javadoc  |   2m 29s |  the patch passed with JDK Private Build-1.8.0_312-8u312-b07-0ubuntu1~20.04-b07  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  |   2m  8s |  tez-api in the patch passed.  |
| +1 :green_heart: |  unit  |   0m 31s |  tez-protobuf-history-plugin in the patch passed.  |
| +1 :green_heart: |  unit  |  64m 46s |  root in the patch passed.  |
| +1 :green_heart: |  asflicense  |   1m  7s |  The patch does not generate ASF License warnings.  |
|  |   | 130m  5s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/11/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/tez/pull/115 |
| JIRA Issue | TEZ-4300 |
| Optional Tests | dupname asflicense shellcheck shelldocs javac javadoc unit xml compile |
| uname | Linux 511efecf49c6 4.15.0-153-generic #160-Ubuntu SMP Thu Jul 29 06:54:29 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | personality/tez.sh |
| git revision | master / a495fb69b |
| Default Java | Private Build-1.8.0_312-8u312-b07-0ubuntu1~20.04-b07 |
| Multi-JDK versions | /usr/lib/jvm/java-11-openjdk-amd64:Ubuntu-11.0.13+8-Ubuntu-0ubuntu1.20.04 /usr/lib/jvm/java-8-openjdk-amd64:Private Build-1.8.0_312-8u312-b07-0ubuntu1~20.04-b07 |
|  Test Results | https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/11/testReport/ |
| Max. process+thread count | 2089 (vs. ulimit of 5500) |
| modules | C: tez-api tez-plugins/tez-protobuf-history-plugin . U: . |
| Console output | https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/11/console |
| versions | git=2.25.1 maven=3.6.3 shellcheck=0.7.1 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

abstractdog commented:
@jteagles : could you please approve this patch?
the installer script has been update according to comments in Jira, now it supports aarch64 too

tez-yetus commented:
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   0m 36s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +0 :ok: |  shelldocs  |   0m  0s |  Shelldocs was not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
| -1 :x: |  test4tests  |   0m  0s |  The patch doesn't appear to include any new or modified tests. Please justify why no new tests are needed for this patch. Also please list what manual steps were performed to verify this patch.  |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   5m 29s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   9m 21s |  master passed  |
| +1 :green_heart: |  compile  |   3m 37s |  master passed with JDK Ubuntu-11.0.13+8-Ubuntu-0ubuntu1.20.04  |
| +1 :green_heart: |  compile  |   3m 12s |  master passed with JDK Private Build-1.8.0_312-8u312-b07-0ubuntu1~20.04-b07  |
| +1 :green_heart: |  javadoc  |   3m 28s |  master passed with JDK Ubuntu-11.0.13+8-Ubuntu-0ubuntu1.20.04  |
| +1 :green_heart: |  javadoc  |   2m 48s |  master passed with JDK Private Build-1.8.0_312-8u312-b07-0ubuntu1~20.04-b07  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 19s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   5m 20s |  the patch passed  |
| +1 :green_heart: |  compile  |   3m 34s |  the patch passed with JDK Ubuntu-11.0.13+8-Ubuntu-0ubuntu1.20.04  |
| +1 :green_heart: |  javac  |   3m 34s |  the patch passed  |
| +1 :green_heart: |  compile  |   3m 14s |  the patch passed with JDK Private Build-1.8.0_312-8u312-b07-0ubuntu1~20.04-b07  |
| +1 :green_heart: |  javac  |   3m 14s |  the patch passed  |
| -1 :x: |  shellcheck  |   0m  0s |  The patch generated 2 new + 0 unchanged - 0 fixed = 2 total (was 0)  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  xml  |   0m  4s |  The patch has no ill-formed XML file.  |
| +1 :green_heart: |  javadoc  |   3m 20s |  the patch passed with JDK Ubuntu-11.0.13+8-Ubuntu-0ubuntu1.20.04  |
| +1 :green_heart: |  javadoc  |   2m 46s |  the patch passed with JDK Private Build-1.8.0_312-8u312-b07-0ubuntu1~20.04-b07  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  |   2m 20s |  tez-api in the patch passed.  |
| +1 :green_heart: |  unit  |   0m 36s |  tez-protobuf-history-plugin in the patch passed.  |
| +1 :green_heart: |  unit  |  70m 14s |  root in the patch passed.  |
| +1 :green_heart: |  asflicense  |   1m 21s |  The patch does not generate ASF License warnings.  |
|  |   | 122m 58s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/13/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/tez/pull/115 |
| JIRA Issue | TEZ-4300 |
| Optional Tests | dupname asflicense shellcheck shelldocs javac javadoc unit xml compile |
| uname | Linux a7c009e00555 4.15.0-58-generic #64-Ubuntu SMP Tue Aug 6 11:12:41 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | personality/tez.sh |
| git revision | master / a495fb69b |
| Default Java | Private Build-1.8.0_312-8u312-b07-0ubuntu1~20.04-b07 |
| Multi-JDK versions | /usr/lib/jvm/java-11-openjdk-amd64:Ubuntu-11.0.13+8-Ubuntu-0ubuntu1.20.04 /usr/lib/jvm/java-8-openjdk-amd64:Private Build-1.8.0_312-8u312-b07-0ubuntu1~20.04-b07 |
| shellcheck | https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/13/artifact/out/diff-patch-shellcheck.txt |
|  Test Results | https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/13/testReport/ |
| Max. process+thread count | 2099 (vs. ulimit of 5500) |
| modules | C: tez-api tez-plugins/tez-protobuf-history-plugin . U: . |
| Console output | https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-115/13/console |
| versions | git=2.25.1 maven=3.6.3 shellcheck=0.7.1 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

jteagles reviewed (approved):
+1. Let's try this out.
